poseidon/poseidonMonitor/test_endPoint.py

```diff
@@ -0,0 +1,27 @@
+"""
+Test module for endPoint.py
+
+Created on 2 October 2017
+@author: Jorissss
+"""
+import pytest
+
+from poseidon.poseidonMonitor import endPoint
+
+test_data = {u'tenant': u'FLOORPLATE',
+             u'mac': u'de:ad:be:ef:7f:12',
+             u'segment': u'prod',
+             u'name': None,
+             u'ip-address': u'102.179.20.100'}
+
+def test_endpoint_creation():
+    endpoint1 = endPoint.EndPoint(test_data)
+    endpoint2 = endPoint.EndPoint.from_json(endpoint1.to_json())
+
+    assert endpoint1.make_hash() == endpoint2.make_hash()
+
```

Comment:
please add another test signature which the states are passed into the constructor.

something like:
endpoint1 = endPoint.EndPoint(test_data,state,next_state)

Reply:
@d-grossman when running 'docker build -f ./Dockerfile -t p2 .' it says all tests pass, when I run 'docker run p2' I get 'ValueError: invalid literal for int() with base 10: 'RABBIT_PORT'' on line 391 in poseidonMonitor.py.

Reply:
you should only need to do the first docker command (build) to see if tests pass.
I erroneously included the 2nd command, please disregard the output of the 2nd command.